builtin/providers/aws/hosted_zones.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+// http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints
+var hostedZoneIDsMap = map[string]string{
+	"us-east-1":      "Z3AQBSTGFYJSTF",
+	"us-west-2":      "Z3BJ6K6RIION7M",
+	"us-west-1":      "Z2F56UZL2M1ACD",
+	"eu-west-1":      "Z1BKCTXD74EZPE",
+	"central-1":      "Z21DNDUVLTQW6Q",
+	"ap-southeast-1": "Z3O0J2DXBE1FTB",
+	"ap-southeast-2": "Z1WCIGYICN2BYD",
+	"ap-northeast-1": "Z2M4EHUR26P7ZW",
+	"sa-east-1":      "Z7KQH4QJS55SO",
+	"us-gov-west-1":  "Z31GFT0UA1I2HV",
+}
```

Comment:
This is unable to be generated from the API json. Attempted to add to aws-sdk in https://github.com/awslabs/aws-sdk-go/pull/222, but that was rejected